main/Helper.cpp

```diff
@@ -1140,6 +1145,117 @@ int SetThreadName(std::thread::native_handle_type thread, const char *name)
 }
 #endif
 
+#if !defined(PTRACE_ATTACH) && defined(PT_ATTACH)
+#  define PTRACE_ATTACH PT_ATTACH
+#endif
+#if !defined(PTRACE_DETACH) && defined(PT_DETACH)
+#  define PTRACE_DETACH PT_DETACH
+#endif
+
+#define _PTRACE(_x, _y) ptrace(_x, _y, NULL, 0)
```

Comment:
OK, good point, it is not needed.
(The idea is to be compatible with both Linux and Mac OSX / BSD, but since this code is not not used for Linux it is not needed)